yjit.rb

```diff
@@ -279,6 +279,10 @@ def _print_stats(out: $stderr) # :nodoc:
       end
       out.puts "num_getivar_megamorphic: " + format_number(13, stats[:num_getivar_megamorphic])
       out.puts "num_setivar_megamorphic: " + format_number(13, stats[:num_setivar_megamorphic])
+      out.puts "num_throw:             " + format_number(13, stats[:num_throw])
+      out.puts "num_throw_break:       " + format_number(13, stats[:num_throw_break])
+      out.puts "num_throw_retry:       " + format_number(13, stats[:num_throw_retry])
+      out.puts "num_throw_return:      " + format_number(13, stats[:num_throw_return])
```

Comment:
You could use `format_number_pct` to format the types of throw as percentages

Reply:
:+1: done https://github.com/ruby/ruby/pull/8188/commits/f155e81d28990cac308d303296224a5558e8ebbb